### PR TITLE
Adjust bitcoin l1 fee rate & convert l1 fee rate to u128

### PR DESCRIPTION
**Pull request by ercecan**
# Description
Adjusts bitcoin l1 fee rate, also l1 fee rate converted to u128 from u64

### CODE REVIEW

File: crates/bitcoin-da/src/service.rs [hunk 1]
```diff
@@ -744,11 +745,10 @@ mod tests {
     async fn check_signature() {
         let da_service = get_service().await;
         let secp = bitcoin::secp256k1::Secp256k1::new();
-        let da_pubkey =
-            Keypair::from_secret_key(&secp, &da_service.sequencer_da_private_key.unwrap())
-                .public_key()
-                .serialize()
-                .to_vec();
+        let da_pubkey = Keypair::from_secret_key(&secp, &da_service.da_private_key.unwrap())
```
Comment (kpp) [inline crates/bitcoin-da/src/service.rs:748]: If da_service.da_private_key is required then it shouldn't be Option anymore

**Review by kpp — APPROVED** [review]

File: crates/bitcoin-da/src/service.rs (hunk 1 above)
Comment (eyusufatik) [reply crates/bitcoin-da/src/service.rs:748]: that's a test + private key should be option, full node does not need to supply that

File: crates/evm/src/tests/genesis_tests.rs [hunk 2]
```diff
@@ -195,6 +195,33 @@ fn genesis_cfg_cancun() {
...
+                Some(reth_rpc_types::BlockNumberOrTag::Earliest),
+                None,
+                &mut working_set,
+            )
+            .unwrap()
+            .unwrap()
+            .header
+            .hash
+            .unwrap();
+        println!("first_block: {:?}", first_block);
```
Comment (eyusufatik) [inline crates/evm/src/tests/genesis_tests.rs:211]: remove prints

Comment (eyusufatik) [reply crates/evm/src/tests/genesis_tests.rs:211]: you dont need this whole block right here i guess

**Review by eyusufatik — CHANGES_REQUESTED** [review]

File: crates/evm/src/tests/genesis_tests.rs [hunk 3]
```diff
@@ -195,6 +195,33 @@ fn genesis_cfg_cancun() {
 #[test]
 fn genesis_block() {
     let (evm, mut working_set) = get_evm(&TEST_CONFIG);
+
+    {
+        let first_block = evm
```
[the remark this answers is not in the document]
Comment (ercecan) [reply crates/evm/src/tests/genesis_tests.rs:200]: oops that's not needed, will remove

**Review by kpp — APPROVED** [review]

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**